tasks/main.yml

```diff
@@ -164,6 +164,14 @@
   become: yes
   template: src=kube-scheduler.yaml dest={{kube_manifests_dir}}/kube-scheduler.yaml
 
+# By the time we get here, kubelet would have been restarting for a while and
+# would have already failed due to the systemd start rate limiting. Since we
+# are specifically waiting below for the apiserver to come up, which requires
+# kubelet, we must force handler execution immediatelly instead of waiting
+# for handler execution to happen at the end of the play, so that kubelet
+# gets successfully started.
+- meta: flush_handlers
+
 - name: wait for apiserver to be up
   command: curl -i --silent http://localhost:8080
```

Comment:
While we are here, can we switch to using the `uri` module instead of `curl`?

Reply:
Hi @naveens, while I can change this to uri easily I don't see a point of changing code that has worked for many years and is expected to be deprecated within next 6 months. I also checked and the new code base already uses uri module for the same check, so we are not carrying this issue further.